src/Umbraco.Core/Composing/ComposerGraph.cs

```diff
@@ -48,8 +45,8 @@ public Composers(IUmbracoBuilder builder, IEnumerable<Type> composerTypes, IEnum
 
         private class EnableInfo
```

Comment:
I think that would be a good suggestion if Enabled/Disabled were ever actually used which I'm not convinced they are.

Goals for this PR were simply 
1. Rename Composers -> ComposerGraph 
1. Mark obsolete IUserComposer

Further effort in cleaning up this class is probably better spent moving towards the pipeline approach on umbraco builder alluded to in https://github.com/umbraco/Umbraco-CMS/discussions/10900#discussioncomment-1238584

I only changed these lines because the field names were causing linting errors but I probably should have left them as they were.